monocle/worker.py

```diff
@@ -786,9 +786,10 @@ def travel_speed(self, point):
 
                 if (normalized not in SIGHTING_CACHE and
                         normalized not in MYSTERY_CACHE):
-                    if (encounter_conf == 'all'
+                    if ((encounter_conf == 'all'
                             or (encounter_conf == 'some'
-                            and normalized['pokemon_id'] in conf.ENCOUNTER_IDS)):
+                            and normalized['pokemon_id'] in conf.ENCOUNTER_IDS))
+                        and self.player_level != None and self.player_level >= 30):
```

Comment:
1. `self.player_level and self.player_level >= 30` sould be fine too, you don't need the check if it is `!= None`
2. I now understand it maybe it would be better this way:
```
if (self.player_level and self.player_level >= 30 and
        ((encounter_conf == 'some' and normalized['pokemon_id'] in conf.ENCOUNTER_IDS)
         or encounter_conf == 'all')):
```
or something like that